services/src/main/scala/cromwell/services/keyvalue/impl/KeyValueDatabaseAccess.scala

```diff
@@ -1,34 +1,7 @@
 package cromwell.services.keyvalue.impl
```

Comment:
FWIW I've got the tall day training tomorrow. 

I've been thinking about CC for most of the morning but that's had the effect of increasing the ways I could see it working, not decreasing. However using the KV seems among the less likely of the reasonable possibilities but who knows